pkg/tfgen/convert_cli_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2016-2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tfgen
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"encoding/json"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	bridgetesting "github.com/pulumi/pulumi-terraform-bridge/v3/internal/testing"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	sdkv2 "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim/sdk-v2"
+	pschema "github.com/pulumi/pulumi/pkg/v3/codegen/schema"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/diag"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/diag/colors"
+)
+
+func TestConvertViaPulumiCLI(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		// Currently there is a test issue in CI/test setup:
+		//
+		// convertViaPulumiCLI: failed to clean up temp bridge-examples.json file: The
```

Comment:
Have we manually tested running the new convert workflow on windows? We do support authoring bridged providers on windows AFAIK.

Reply:
Nope, we haven't. It's not on the critical path. I can add it to the bottom of the epic.

Reply:
It's necessary for removing the old convert logic, but shouldn't be blocking for us to use the new logic. We might want to add a warning for users who turn on the new converter on windows computers.